cookbook/agent_os/rbac/symmetric/basic.py

```diff
@@ -0,0 +1,120 @@
+"""
+Basic RBAC Example with AgentOS
+
+This example demonstrates how to enable RBAC (Role-Based Access Control)
+with JWT token authentication in AgentOS using middleware.
+
+Prerequisites:
+- Set JWT_VERIFICATION_KEY environment variable or pass it to middleware
+- Endpoints are automatically protected with default scope mappings
+"""
+
+import os
+from datetime import UTC, datetime, timedelta
+
+import jwt
+from agno.agent import Agent
+from agno.db.postgres import PostgresDb
+from agno.models.openai import OpenAIChat
+from agno.os import AgentOS
+from agno.os.config import AuthorizationConfig
+from agno.tools.duckduckgo import DuckDuckGoTools
+
+# JWT Secret (use environment variable in production)
+JWT_SECRET = os.getenv("JWT_VERIFICATION_KEY", "your-secret-key-at-least-256-bits-long")
+
+# Setup database
+db = PostgresDb(db_url="postgresql+psycopg://ai:ai@localhost:5532/ai")
+
+# Create agents
+research_agent = Agent(
+    id="research-agent",
+    name="Research Agent",
+    model=OpenAIChat(id="gpt-4o"),
+    db=db,
+    tools=[DuckDuckGoTools()],
+    add_history_to_context=True,
+    markdown=True,
+)
+
+# Create AgentOS
+agent_os = AgentOS(
+    id="my-agent-os",
+    description="RBAC Protected AgentOS",
+    agents=[research_agent],
+    authorization=True,
```

Comment:
correct me if i'm wrong but the `README` suggests (`When `verify_audience=True` (default when authorization is enabled), tokens with a mismatched `aud` claim will be rejected with a 401 error.`) but there is no way to set `verify_audience` when using `authorization=True` via `AgentOS` defaulting it to False? 

Should we have it in the `authorization_config`?